packages/blockchain-utils-linking/src/__tests__/ethereum.test.ts

```diff
@@ -162,4 +162,31 @@ describe('EthereumAuthProvider', () => {
     const auth = new ethereum.EthereumAuthProvider(provider, address)
     await expect(auth.withAddress(address).createLink(testDid)).resolves.toMatchSnapshot()
   })
+
+  describe('various providers', () => {
+    test('sendAsync', async () => {
```

Comment:
it would be nice to use `jest.spyOn` to spy on the three send methods (send, sendAsync, request) and then assert in the tests that the right one that you expected gets called and not the other ones

Reply:
👍